tests/storage-vm

```diff
@@ -457,7 +322,7 @@ for poolDriver in $poolDriverList; do
 
                 echo "==> Checking VM config disk filesystem is not XFS"
                 serverPID="$(lxc query /1.0 | jq .environment.server_pid)"
-                nsenter -m -t "${serverPID}" findmnt --noheadings --output=FSTYPE --mountpoint /var/snap/lxd/common/lxd/devices/v1/config.mount | grep -xF ext4
+                [ "$(nsenter -m -t "${serverPID}" findmnt --noheadings --output=FSTYPE --mountpoint /var/snap/lxd/common/lxd/devices/v1/config.mount)" = ext4 ]
```

Comment:
As a matter of style, please quote the right hand side of the comparison as well. Please also do so for all new occurrences, thanks!